### ISSUE TITLE
Allow developers to set a customer "published_at" date for Addons and Versions

### ISSUE THREAD

**Issue by scottrobertson** [romhut/romhut.com#9]
This depends on #8

**Comment by MatthewBooth** [comment]
Is this the same as setting a future publishing date? Such that you can say "publish this ROM in 3 hours" . Or is this different?

**Comment by scottrobertson** [comment]
You should genuinely be a project manager.

**Comment by MatthewBooth** [comment]
I'll assume that's different.... opening a new issue :D

**Comment by scottrobertson** [comment]
1 issue I am coming across at the moment is that when you click "publish" in the dashboard on a Version or Addon, we update the published_at datetime on the Rom.

 If we allow devs to set the published_at time in the future on Versions or Addons then the Rom published_at time will be out of sync. We cannot set the roms published_at to the future because it will mess all of the listings up on the front end of the site.

**Comment by scottrobertson** [comment]
For now, this is not going to happen.
